Documentation build is broken

The build of the docs has some errors (and warnings) ..

````
$ make clean docs-live
CLEAN     docs-clean
PYENV     usage: source ./local/py3/bin/activate
build searx/brand.py
build utils/brand.env
PYENV     install .
...
````
1. Error comes from pip ...
````
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.
````
2. is a dependency error ..
````
readme-renderer 26.0 requires Pygments>=2.5.1, but you'll have pygments 2.1.3 which is incompatible.
````

3. I guess comes from #2112

````
ERROR: centos-: uWSGI not yet implemented
````
4. error is from sphinx-autobuild, which has changed the command line (solved by #2210)

````
PYENV     installing Sphinx
SPHINX    docs-live --> file:///share/test123/searx/dist/docs
usage: sphinx-autobuild [-h] [--port PORT] [--host HOST]
                        [--re-ignore RE_IGNORE] [--ignore IGNORE]
                        [--no-initial] [--open-browser] [--delay DELAY]
                        [--watch DIR] [--pre-build COMMAND] [--version]
                        sourcedir outdir [filenames [filenames ...]]
sphinx-autobuild: error: unrecognized arguments: --poll -B
make: *** [Makefile:88: docs-live] Fehler 2
````

--- a/Makefile
+++ b/Makefile
@@ -80,11 +80,11 @@ run:  buildenv pyenvinstall
 sphinx-doc-prebuilds:: buildenv pyenvinstall prebuild-includes
 
 PHONY += docs
-docs:  sphinx-doc-prebuilds sphinx-doc
+docs:  sphinx-doc-prebuilds
 	$(call cmd,sphinx,html,docs,docs)
 
 PHONY += docs-live
-docs-live:  sphinx-doc-prebuilds sphinx-live
+docs-live:  sphinx-doc-prebuilds
 	$(call cmd,sphinx_autobuild,html,docs,docs)
 
 PHONY += prebuild-includes

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -12,9 +12,13 @@ selenium==3.141.0
 twine==3.2.0; python_version >= "3.6"
 twine==1.15.0; python_version < "3.6"
 Pallets-Sphinx-Themes==1.2.3
-Sphinx==3.0.1
+Sphinx==3.2.1; python_version >= '3.6'
+Sphinx==3.0.1; python_version < '3.6'
 sphinx-issues==1.2.0
 sphinx-jinja==1.1.1
-sphinx-tabs==1.1.13
+sphinx-tabs==1.3.0; python_version >= '3.6'
+sphinx-tabs==1.1.13; python_version < '3.6'
 sphinxcontrib-programoutput==0.16
+sphinx-autobuild==2020.9.1; python_version >= '3.6'
+sphinx-autobuild==0.7.1; python_version < '3.6'
 linuxdoc @ git+http://github.com/return42/linuxdoc.git@70673dcf69e705e08d81f53794895dc15c4920b3#egg=linuxdoc

--- a/utils/makefile.sphinx
+++ b/utils/makefile.sphinx
@@ -21,23 +21,6 @@ else
   SPHINX_VERBOSE =
 endif
 
-## SPHINXVERS variable
-## ===================
-##
-## .. _requirement-specifiers: https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers
-##
-## Sphinx version to use, when building documentation.  Set this when calling
-## build target.  The default value is empty (install latest), to select a
-## specific version use a requirement-specifiers_.  E.g. to build your target
-## 'doc' with a select sphinx-doc_ version 1.7.9::
-##
-##     make SPHINXVERS='==1.7.9' docs
-##
-## To build with latest 1.7::
-##
-##     make SPHINXVERS='>=1.7,<1.8' docs
-##
-SPHINXVERS  ?=
 
 docs-help:
 	@echo  'makefile.sphinx:'
@@ -55,17 +38,6 @@ docs-help:
 # ------------------------------------------------------------------------------
 # requirements
 # ------------------------------------------------------------------------------
-
-sphinx-doc-prebuilds:: $(PY_ENV)
-
-sphinx-doc: sphinx-doc-prebuilds
-	@echo "PYENV     installing Sphinx$(SPHINXVERS)"
-	$(Q)$(PY_ENV_BIN)/pip install $(PIP_VERBOSE) 'Sphinx$(SPHINXVERS)'
-
-sphinx-live: sphinx-doc-prebuilds
-	@echo "PYENV     installing Sphinx$(SPHINXVERS)"
-	$(Q)$(PY_ENV_BIN)/pip install $(PIP_VERBOSE) 'Sphinx$(SPHINXVERS)' sphinx-autobuild
-
 
 PHONY += msg-texlive texlive
 
@@ -107,7 +79,7 @@ quiet_cmd_sphinx = SPHINX    $@ --> file://$(abspath $(DOCS_DIST)/$5)
 	-b $2 -c $3 -d $(DOCS_BUILD)/.doctrees $4 $(DOCS_DIST)/$5
 
 quiet_cmd_sphinx_autobuild = SPHINX    $@ --> file://$(abspath $(DOCS_DIST)/$5)
-      cmd_sphinx_autobuild = PATH="$(PY_ENV_BIN):$(PATH)" $(PY_ENV_BIN)/sphinx-autobuild  $(SPHINX_VERBOSE) --poll -B --host 0.0.0.0 --port 8080 $(SPHINXOPTS)\
+      cmd_sphinx_autobuild = PATH="$(PY_ENV_BIN):$(PATH)" $(PY_ENV_BIN)/sphinx-autobuild  $(SPHINX_VERBOSE) --open-browser --host 0.0.0.0 --port 8080 $(SPHINXOPTS)\
 	-b $2 -c $3 -d $(DOCS_BUILD)/.doctrees $4 $(DOCS_DIST)/$5
 
 quiet_cmd_sphinx_clean = CLEAN     $@

